app.js

```diff
@@ -74,6 +75,8 @@ app.use(methodOverride((req, res, next) => { // for POST requests
     }
 }));
 
+// expose font awesome font to the public
+app.use('/fonts/fontawesome', express.static(__dirname + '/node_modules/font-awesome/fonts'));
```

Comment:
ich habe das über webpack gelöst, siehe https://github.com/schul-cloud/schulcloud-client/pull/845/files

Reply:
Da ist jetzt die Frage, was besser ist. @Michael-Janke will von gulp weg kommen und du schlägst vor mehr Features nach gulp zu verlagern. Ich bin da neutral, aber ihr müsstet euch einigen 😅 Finde deine Idee hier aber super und ist vermutlich auch performanter :)

Reply:
Webpack hat den Vorteil, dass man sich darum nie wieder kümmern muss und es vor allem inkrementelle Builds unterstützt. (theoretisch)
Ist aber jetzt viel mehr Arbeit und wirft andere Probleme auf, weil vieles angepasst werden muss.

Ist halt ziemlich unsexy, dass ich wenn ich ein neues module einbinde, dass ich an die gulp-Datei ran muss. 

Auch wird zur Zeit noch gulp Version 3 verwendet. Gulp ist längst bei Version 4, welche performanter ist und vieles von Haus aus macht.

Das ist aber nicht eine Entscheidung die wir jetzt treffen können, sondern in großer Runde getroffen werden sollte 😄